docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,77 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after a configuration update
+
+The update of an existing Elasticsearch cluster configuration can fail because the operator is unable to apply the changes required while replacing the pods of a given Elasticsearch cluster.
+A key indicator is when the Phase of the Elasticsearch resource is in `ApplyingChanges` state for too long:
+
+[source,sh]
+----
+kubectl get es
```

Comment:
What is the correct format to display both output and command at the same time?  We have a few different methods in the [Troubleshooting](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-troubleshooting.html) doc section.  Right now we have two different formats for this new section.

Existing formats
- Single command (no leading $, output in separate block) in [common](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-common-problems.html) and [methods](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-troubleshooting-methods.html) 
    ```bash
    kubectl patch sts elastic-operator....
    ```
- Command with output (with leading $) in [common](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-common-problems.html)
    ```bash
    $ kubectl get secret quickstart-es-elastic-user -o yaml
    apiVersion: v1
    ...
    ```
- Command with output (without the leading $ and newline between input and output) in [methods](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-troubleshooting-methods.html)
    ```
    kubectl get pods
    
    NAME                                 READY     STATUS    RESTARTS   AGE
    ...
    ```

Reply:
Let's remove the leading $.